docs/MIGRATION-ESLINT.md

```diff
@@ -0,0 +1,104 @@
+# Migration from TSLINT to ESLINT
+
+## install eslint schematics
+
+please use `gitbash` to execute commands.
+
+```bash
+npx ng add @angular-eslint/schematics
+```
+
+## run the schematics
+
+```bash
+npx ng g @angular-eslint/schematics:convert-tslint-to-eslint <project-name>
+```
+
+## Add configuration
+
+### package.json
+
+```json
+{
+  ...
+  "devDependencies": {
+    "@nationalbankbelgium/eslint-config": "^12.0.0",
+    ...
+    "eslint-config-prettier": "^8.8.0",
```

Comment:
Could we recommend to directly use the current latest version `9.1.0` ?

Reply:
I've tested in angular 16 but not in previous versions so i added in angular 16